Dockerfile

```diff
@@ -1,4 +1,15 @@
 FROM registry.access.redhat.com/rhel7-atomic
+MAINTAINER Portworx Inc. <support@portworx.com>
+
+LABEL name="openstorage/stork" \
+      maintainer="support@portworx.com" \
+      vendor="Portworx Inc." \
+      version="1.0.2" \
+      release="1" \
+      summary="STORK" \
+      description="Storage Orchestrator Runtime for Kubernetes" \
+      url="https://github.com/libopenstorage/stork" \
+      io.openshift.tags="portworx,storage,pv,pvc,storageclass,stork,persistent,openstorage"
```

Comment:
Recommend to add the following:

```
+      summary="Storage Orchestrator Runtime for Kubernetes" \
+      description="Storage Orchestrator Runtime for Kubernetes" \
+      io.k8s.description="Storage Orchestrator Runtime for Kubernetes" \
+      io.k8s.display-name="STORK" \
```

Otherwise Kubernetes adds its defaults (can verify via `docker inspect <image>`).